### ISSUE TITLE
[Graphics.hx] 'default' is a Haxe keyword, needs to be replaced to prevent compile errors

### ISSUE THREAD

**Issue by kevansevans** [excessive/lovr-haxe-wrappergen#5]
On line 86 of the generated Graphics.hx, the following line causes a compile failure in Haxe:

``@:overload(function (default: DefaultShader, ?options: lua.Table<Dynamic, Dynamic>): Shader {})``

``default`` is a reserved keyword to describe getter/setter behavior and switch statement fallback in Haxe. Easiest solution to this would be to generate a different argument name, changing it to ``_default`` solved the issue for me.

**Comment by shakesoda** [comment]
yeah, there are a few things that manually need to be fixed in the output right now. i've ended up with manually maintained bindings based on the generated ones for a while... need to do some updates here.
